Source/VMware.vSphereDSC/DSCResources/VMHostNetwork/VMHostVss/VMHostVss.ps1

```diff
@@ -107,15 +93,14 @@ class VMHostVss : VMHostBaseDSC {
     [VMHostVss] Get() {
         Write-Verbose -Message "$(Get-Date) $($s = Get-PSCallStack; "Entering {0}" -f $s[0].FunctionName)"
 
-        $this.ConnectVIServer()
-        $vmHost = $this.GetVMHost()
-
         $result = [VMHostVss]::new()
         $result.Server = $this.Server
-        $result.Name = $vmHost.Name
 
+        $this.ConnectVIServer()
+        $vmHost = $this.GetVMHost()
+        $this.GetNetworkSystem($vmHost)
```

Comment:
Add a new line between $this.GetNetworkSystem($vmHost)  and $result.Name = $vmHost.Name for better readability.

Reply:
The new line is still missing here.

Reply:
Changed